src/models.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import os
+import shutil
+import subprocess
+
+from pydantic import BaseModel, Field
+
+
+class TraversalRecord(BaseModel):
+    """_description_
+
+    Args:
+        BaseModel (_type_): _description_
+    """
+    base_path: str
+    env_path: str
+    chart_dir: str
+    rendered_path: str
+    name: str = Field(init=True)
+
+    def helm_exec_string(self):
+        return f"helm template {self.name} {self.chart_dir} -f {self.base_path} -f {self.env_path} --output-dir {self.rendered_path}"
```

Comment:
why chart_dir and base_path, I think it's more correct to name it:
`chart_name, chart_path`